requirements.txt

```diff
@@ -4,3 +4,9 @@ sphinx_rtd_theme==1.0.0
 git+https://github.com/pdavide/sphinxcontrib-discourse
 attrs==21.4.0
 glossarpy==0.1.2
+sphinxcontrib-applehelp==1.0.4
```

Comment:
Disclaimer, not a Python expert, but I was looking -- shouldn't we do a `pip freeze` to pin all dependencies? This will fix the immediate problem, but without the freeze, couldn't a variant of the problem might creep back in?

Reply:
pip freeze just generates a requirements file based on the currently installed packages in a given environment. Agreed with the above that we should just upgrade when its expedient (and then pin the packages to a known working version of each).

Reply:
Presumably we didn't do a `pip freeze` before, or this wouldn't have come up?

Reply:
I committed a new `requirements.txt` that freezes all dependencies.  The process was to create a clean virtual env, run `pip install -r requirements.txt`, confirm a successful build, and generate a new `requirements.txt` via `pip freeze`.